packages/config/config/devices/0x001d/zw4sf.json

```diff
@@ -0,0 +1,112 @@
+// Leviton ZW4SF
+// 4 Speed Fan Controller
+{
+	"manufacturer": "Leviton",
+	"manufacturerId": "0x001d",
+	"label": "ZW4SF",
+	"description": "Decora Smart 4 Speed Fan Controller",
```

Comment:
Lose "Decora Smart." If you really want to keep it, fix the comment above to match.